mono/tests/tailcall-virt.fs

```diff
@@ -0,0 +1,27 @@
+// Test tail.callvirt.
```

Comment:
This file is not used anywhere and thus useless. Please remove.

Reply:
It is the basis of tailcall.cs, as the comments say.